packages/sdk-core/CHANGELOG.md

```diff
@@ -29,6 +31,19 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Internal
 - Use `eslint-plugin-prettier` over separate `prettier` instance
 
+### Breaking
+- The `SuperToken` class is now an abstract base class and no longer contains the functions `upgrade` and `downgrade`.
+- `underlyingToken` is possibly undefined on `SuperToken`: `WrapperSuperToken` has `underlyingToken`, but `PureSuperToken` and `NativeAssetSuperToken` do not.
+> NOTE: These changes are due to the split of `SuperToken` into `WrapperSuperToken`, `PureSuperToken` and `NativeAssetSuperToken` classes.
+  - Migration: if you are using TypeScript, you will need to specify the type of token you are attempting to load:
```

Comment:
No, you don't...?

Reply:
I had this issue in testing dawg

Reply:
What issue? In `sdk-redux`, we never know which type it is so we do `instanceof` instead.

Reply:
the issue is that it complains because underlyingToken might be undefined, so when I access it I get a TypeError